Can't build documentation on stable Rust

If I try building documentation for a crate that depends on owo-colors on Rust 1.57, I get:

```
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> /opt/cargo/registry/src/github.com-1ecc6299db9ec823/owo-colors-3.1.0/src/lib.rs:72:37
   |
72 | #![cfg_attr(all(doc, not(doctest)), feature(doc_cfg))]
   |                                     ^^^^^^^^^^^^^^^^
```

I think this might need to be gated to docs-rs somehow?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,6 +13,7 @@ categories = ["command-line-interface"]
 
 [package.metadata.docs.rs]
 all-features = true
+rustdoc-args = ["--cfg", "doc_cfg"]
 
 [[example]]
 name = "supports_color"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -69,7 +69,7 @@
 //! println!("{}", text.style(my_style));
 //! ```
 #![cfg_attr(not(test), no_std)]
-#![cfg_attr(all(doc, not(doctest)), feature(doc_cfg))]
+#![cfg_attr(doc_cfg, feature(doc_cfg))]
 #![doc(html_logo_url = "https://jam1.re/img/rust_owo.svg")]
 #![warn(missing_docs)]
 

